app/javascript/packages/verify-flow/steps/password-confirm/index.ts

```diff
@@ -0,0 +1,10 @@
+import { t } from '@18f/identity-i18n';
+import type { FormStep } from '@18f/identity-form-steps';
+import type { VerifyFlowValues } from '../..';
+import form from './password-confirm-step';
+
+export default {
+  name: 'password_confirm',
+  title: t('titles.idv.session.review'),
```

Comment:
Via build failure, this locale key doesn't exist. I think it should be....

```suggestion
  title: t('idv.titles.session.review'),
```